### ISSUE TITLE
Feature for adding a folder specific share box.

### ISSUE THREAD

**Issue by mateosalta** [nitroshare/nitroshare-desktop#10]
In the last version you could change where files went. It would be cool to have a special share box that direct the transfer to a specific folder. Quick mock-up:
![screenshot from 2015-01-30 11 43 17](https://cloud.githubusercontent.com/assets/1965279/5980598/b1cc369a-a877-11e4-9a1e-3a219ae6f5de.png)

Use cases I'm thinking of are like offices, where on desks you used to have In/Out paper trays for important files, you could setup a specific folder for those uses, or as well have a diffrent folder for video/pictures ect.

Or if there is a central server, you could orginize to a few diffrent folders on the same location.

Basically multiple targest for the same computer.

For security reasons this migh be an *intresting* thing, but maybe a "authorized" folder list on the Recieving machine might make it ok.

**Comment by nathan-osman** [comment]
That's a great idea. Once share boxes are re-implemented, they'll have two properties:

- the target device
- the target directory on the device

**Comment by rapgro** [comment]
That's tricky. You should let choose from accessible targets and with enough free space. It may be easy to pollute disk of target device and even let them run out of space.

**Comment by rapgro** [comment]
In latest version 0.3.4, it's not possible to automatically create download folder. Even the suggestion is not there, if you don't change any default setting.
